pkg/rsyslog/rsyslog.conf

```diff
@@ -17,19 +15,22 @@ $FileCreateMode 0640
 $DirCreateMode 0755
 $Umask 0022
 
+main_queue(
+  queue.size="1000000"
+  queue.type="Disk"
+  queue.fileName="main_edge_node_log_queue"
+  queue.maxDiskSpace="2g"
```

Comment:
All these #s are fine for now, but I think we need to start thinking about keying them off of EVE config.

Reply:
I'll create a tracker story for this.

Reply:
https://www.pivotaltracker.com/story/show/170641589
This is the story I created on pivotal tracker.